app.js

```diff
@@ -1,5 +1,6 @@
 import Music from "./music.js";
 import PERMANENT from "./world.js";
+import { pushToLoki } from "./telemetry.js";
```

Comment:
@lxkarp Looks like telemetry.js didn't get added to the commit, add and re-upload?

Reply:
Done, thanks!